front-end/package.json

```diff
@@ -37,7 +37,7 @@
         "@popperjs/core": "^2.10.2",
         "bootstrap": "^5.1.3",
         "class-transformer": "^0.5.1",
-        "fecfile-validate": "https://github.com/fecgov/fecfile-validate#0bad1ff320746cd0d457e0094b15511b95384df5",
+        "fecfile-validate": "https://github.com/fecgov/fecfile-validate#e692949fc4d116a852e7d0fe4d7563d94518e234",
```

Comment:
should this point to [64457d0](https://github.com/fecgov/fecfile-validate/pull/77/commits/64457d0be3a8fa50193c98b5e48af53a87984311) since you've made changes to the validator?